samples/nrf9160/cell_pos/src/main.c

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <modem/lte_lc.h>
+#include <net/cloud.h>
+#include <net/socket.h>
+#include <net/nrf_cloud_cell_pos.h>
+#include <dk_buttons_and_leds.h>
+#include <power/reboot.h>
+
+#include <logging/log.h>
+
+#define SERVICE_INFO_GPS \
+	"{\"state\":{\"reported\":{\"device\": {\"serviceInfo\":{\"ui\":[\"GPS\"]}}}}}"
+
+LOG_MODULE_REGISTER(scell, CONFIG_CELL_POS_SAMPLE_LOG_LEVEL);
+
+static struct cloud_backend *cloud_backend;
+static struct k_delayed_work scell_work;
+static struct k_delayed_work reboot_work;
+
+static void cloud_send_msg(void)
+{
+	int err;
+
+	LOG_DBG("Publishing message: %s", log_strdup(CONFIG_CLOUD_MESSAGE));
+
+	struct cloud_msg msg = {
+		.qos = CLOUD_QOS_AT_MOST_ONCE,
+		.endpoint.type = CLOUD_EP_MSG,
+		.buf = CONFIG_CLOUD_MESSAGE,
+		.len = sizeof(CONFIG_CLOUD_MESSAGE)
+	};
+
+	err = cloud_send(cloud_backend, &msg);
+	if (err) {
+		LOG_ERR("cloud_send failed, error: %d", err);
+	}
+}
+
+static void scell_work_fn(struct k_work *work)
+{
+	int err;
+
+	ARG_UNUSED(work);
+
+	err = nrf_cloud_request_cell_location(CELL_LOC_TYPE_SINGLE, true);
+
+	if (err) {
+		LOG_ERR("nRF Cloud SCELL request failed, error: %d", err);
+	}
+
+}
+
+static void send_service_info(void)
+{
+	int err;
+	struct cloud_msg msg = {
+		.qos = CLOUD_QOS_AT_MOST_ONCE,
+		.endpoint.type = CLOUD_EP_STATE,
+		.buf = SERVICE_INFO_GPS,
+		.len = strlen(SERVICE_INFO_GPS)
+	};
+
+	err = cloud_send(cloud_backend, &msg);
+	if (err) {
+		LOG_ERR("Failed to send message to cloud, error: %d", err);
+		return;
+	}
+
+	LOG_INF("Service info sent to cloud");
+}
+
+static void cloud_event_handler(const struct cloud_backend *const backend,
+				const struct cloud_event *const evt,
+				void *user_data)
+{
+	int err;
+
+	ARG_UNUSED(backend);
+	ARG_UNUSED(user_data);
+
+	switch (evt->type) {
+	case CLOUD_EVT_CONNECTING:
+		LOG_INF("CLOUD_EVT_CONNECTING");
+		break;
+	case CLOUD_EVT_CONNECTED:
+		LOG_INF("CLOUD_EVT_CONNECTED");
+		break;
+	case CLOUD_EVT_READY:
+		LOG_INF("CLOUD_EVT_READY");
+		/* Update nRF Cloud with GPS service info that device has GPS capabilities */
```

Comment:
suggest:
`/* Inform nRF Cloud that this device has GPS capabilities */`

Reply:
Removed since we don't do so now

Reply:
Adding the service info should stay, without it nrfcloud.com will not display the map card.  It will eventually display the approximate location from scell and mcell requests.

Reply:
OK, will add it back